.github/workflows/e2e.yml

```diff
@@ -27,22 +27,10 @@ jobs:
     if: needs.pre_job.outputs.should_skip != 'true'
 
     steps:
-
-      - name: Checkout code
-        uses: actions/checkout@v3
+      - uses: actions/checkout@v3
         with:
           submodules: 'recursive'
-
-      - name: Setup Node
-        uses: ./.github/actions/node
-
-      - name: Install Foundry
-        uses: foundry-rs/foundry-toolchain@v1
-        with:
-          version: nightly
-
-      - name: Setup Testing Environment
-        run: ./e2e/sbin/setup.sh
-
-      - name: Run Test
-        run: ./e2e/sbin/test.sh ${{ matrix.test-name }}
+      - uses: ./.github/actions/node
+      - run: make
```

Comment:
add blank lines between steps

Reply:
fyi its not generally required https://docs.github.com/en/actions/quickstart